corehq/apps/app_manager/dbaccessors.py

```diff
@@ -22,14 +22,14 @@ def domain_has_apps(domain):
     return len(results) > 0
 
 
-def get_latest_released_app_doc(domain, app_id, min_version=None):
+def get_latest_released_app_doc(domain, app_id):
```

Comment:
I'm pretty sure this is never relevant.  `min_version` is only ever set
if the app_id passed in corresponds to a released app, so there can't be
a situation where the latest released app has an ealier version, since
the set of released app versions *contains* the passed in app.